synapse/api/room_versions.py

```diff
@@ -58,7 +58,11 @@ class RoomVersion(object):
     enforce_key_validity = attr.ib()  # bool
 
     # bool: before MSC2261/MSC2432, m.room.aliases had special auth rules and redaction rules
-    special_case_aliases_auth = attr.ib(type=bool, default=False)
+    special_case_aliases_auth = attr.ib(type=bool)
+
+    # bool: MSC2209: Check 'notifications' key while verifying
+    # m.room.power_levels auth rules.
+    check_notifications_auth = attr.ib(type=bool)
```

Comment:
I think this would be better called something like `limit_notifications_power_levels`

Reply:
That is likely better. I'll update it.